spec/pg/gc_compact_spec.rb

```diff
@@ -0,0 +1,82 @@
+# -*- rspec -*-
+# encoding: utf-8
+
+require_relative '../helpers'
+
+if GC.respond_to?(:compact)
+	describe "GC.compact" do
```

Comment:
You may be able to indent less using RSpec's [built-in filtering](https://relishapp.com/rspec/rspec-core/docs/filtering/conditional-filters).

```suggestion
describe "GC.compact", if: GC.respond_to?(:compact) do
```

Reply:
The filtering is a bit weird though as it still executes the block.
Probably fine here since all the logic is in `before`/`it`.

Reply:
`describe` blocks are still executed in case of `if: false`. Only the execution of `it` blocks is omitted.

Reply:
I moved all code into before, it and after blocks and made use of the :if filtering. Thanks for the hint!